### ISSUE TITLE
ui: Add smooth scrolling

### ISSUE THREAD

**Issue by seanmrnda** [fynelabs/nomad#73]
When you scroll the main window of the app it feels a bit sluggish.

**Comment by andydotxyz** [comment]
Can you share which device this is on? I wonder if it is a duplicate of fyne-io/fyne#1062

**Comment by schwrzstrbn** [comment]
> Can you share which device this is on? I wonder if it is a duplicate of [fyne-io/fyne#1062](https://github.com/fyne-io/fyne/issues/1062)

OS: Android 10
RAM: 2gb
MODEL: Realme C2

**Comment by schwrzstrbn** [comment]
> Can you share which device this is on? I wonder if it is a duplicate of [fyne-io/fyne#1062](https://github.com/fyne-io/fyne/issues/1062)

OS: Android 10
RAM: 2gb
MODEL: Realme C2

**Comment by andydotxyz** [comment]
Yeah, it's a duplicate then. Smooth scrolling is part of the toolkit not the app, when it's fixed upstream we get the fix here :)
